vars/capsuleSystemTests.groovy

```diff
@@ -0,0 +1,196 @@
+/* groovylint-disable 
+  BuilderMethodWithSideEffects, CompileStatic, DuplicateStringLiteral, 
+  FactoryMethodName, VariableTypeRequired */
+
+
+void gitClone(String repository, String branch, String directory, String credentialsId) {
+  dir(directory) {
+    checkout([
+      $class: 'GitSCM',
+      branches: [[name: branch]],
+      userRemoteConfigs: [[
+          url: 'git@github.com:vegaprotocol/' + repository + '.git',
+          credentialsId: credentialsId
+    ]]])
+  }
+}
+
+void buildGoBinary(String directory, String outputBinary, String packages) {
+  dir(directory) {
+    sh 'go mod vendor'
+    sh 'go build -o ' + outputBinary + ' ' + packages
+  }
+}
+
+void trysh(String shellScript) {
+  try {
+    sh shellScript
+  } catch(err) {
+  }
+}
+
+void cleanup(String testDirectoryPath) {
+  trysh 'rm ' + testDirectoryPath + '/vegacapsule'
+  trysh 'rm ' + testDirectoryPath + '/vega'
+  trysh 'rm ' + testDirectoryPath + '/data-node'
+  trysh 'rm -r ' + testDirectoryPath + '/testnet'
+  trysh 'rm ' + testDirectoryPath + '/vegawallet'
+  trysh 'rm ' + testDirectoryPath + '/config_system_tests.hcl'
+}
```

Comment:
What about using loop and bash safe failover instead of `trysh` ?
```
['vegacapsule', 'vega', 'data-node', 'testnet', 'vegawallet', 'config_system_tests.hcl'].each { sh "rm -rf ${testDirectoryPath}/${it} || true" }
```

Reply:
Hmmm I can, that's probably better than my copy-paste